src/Assets/AssetContainer.php

```diff
@@ -140,6 +140,10 @@ public function url()
 
         $url = rtrim($this->disk()->url('/'), '/');
 
+        if (str_starts_with($url, config('app.url'))) {
+            $url = substr($url, strlen(config('app.url')));
+        }
+
```

Comment:
why not `Str::removeLeft` here?

Reply:
@edalzell I only found `Str::chopStart()` but that's not available in Laravel 10, so I did it the traditional way but if that's available... sure! Much nicer. Will check... Should have ask my coworker.

Reply:
You certainly should have. It's a Statamic function not Laravel, they extend Laravel's Str.

Reply:
@edalzell done